content/en/docs/concepts/distributions.md

```diff
@@ -62,6 +74,12 @@ If you are building your own distribution, the [OpenTelemetry Collector
 Builder](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder)
 might be a good starting point.
 
+### Language Specific Instrumentation libraries
+
+There are language specific extensibility mechanisms to customize the instrumentation libraries:
+
+- [Java](../../instrumentation/java/extensibility)
```

Comment:
this extensibility is specific to the Javaagent, there are different extensibility mechanisms for (what we call) "library" instrumentation, I'm not sure the best way to call this out though

Reply:
Understood, so if someone is using the SDK without the Agent, they will need to find another way to extensibility? Does something light this exist?

To add a little bit where I am coming from: As a vendor having a clear contract to extend on top of is highly valuable, because it takes away some uncertainty if things stay that way or change. Also there are multiple ways per language on how to do things, so having a "best practice" is great.  

Finally, _having a way to  add new features and capabilities to the agent without having to create a separate distribution_ [[1](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/examples/extension/README.md)] is adding value for the end user: instead of replacing distributions they only need to swap out extensions (or MAYBE can use extensions from multiple vendors).

Reply:
When not using the agent, I would say the "best practice" is to use the [autoconfigure module](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#opentelemetry-sdk-autoconfigure).

There are further extensibility points for (what we call) library instrumentation (but which still fall under otel "auto-instrumentation" definition). These allow you to customize individual instrumentation (as opposed to the autoconfigure module which allows you to customize the SDK). These library instrumentation extensibility points are not well documented yet though.

Reply:
@trask, thanks. I can update `Java` here with `Java Agent`. Or what is the best name for the agent?

Reply:
> what is the best name for the agent?

we have been using `Javaagent` (one word), to try to differentiate from common usage of `Agent` meaning out-of-proc agent

Reply:
thx, put Javaagent.